examples/ota-provider-app/linux/main.cpp

```diff
@@ -16,13 +16,13 @@
  *    limitations under the License.
  */
 
+#include <app/clusters/ota-provider/DefaultOTAProviderUserConsentProvider.h>
```

Comment:
Is the Provider at the end intentional or redundant?

```suggestion
#include <app/clusters/ota-provider/DefaultOTAProviderUserConsent.h>
```